it/uk/gov/hmrc/transitmovements/repositories/MovementsRepositorySpec.scala

```diff
@@ -195,23 +195,37 @@ class MovementsRepositorySpec
       )
     }
 
+  it should "return a list of movement responses for the supplied EORI if there are movements that were updated since the given time" in {
```

Comment:
can we not combine arrival and departure cases using foreach? to make test cases more compact. instead of writing  separately for arrival and departure. as the code is common should be written using foreach if possible

Reply:
Yes, tried to implement this, looks like mongo db is getting into dup key issues - better we leave as is